app/javascript/dashboard/routes/dashboard/settings/inbox/channels/Website.vue

```diff
@@ -182,3 +210,12 @@ export default {
   },
 };
 </script>
+
+<style scoped>
+.greetings--richtext {
+  padding: 0 var(--space-normal);
+  border-radius: var(--border-radius-normal);
+  border: 1px solid #e0e6ed;
+  margin: 0 0 1.6rem;
```

Comment:
We can use Css variable here also.

Reply:
Replaced in [commit](https://github.com/chatwoot/chatwoot/pull/2952/commits/246678c6afdd7d620d2eefd1c91667919f90871a)